terraform/layer0.tf

```diff
@@ -52,11 +52,6 @@ variable "ssl_certificate" {
   description = "SSL certificate name for the web load balancer"
 }
 
-variable "wait" {
-  description = "Wait for the service deployment to complete"
-  default     = false
-}
-
 # Resources
 
 # todo: make name configurable
```

Comment:
why don't we knock out these TODOs in this PR?

Reply:
Will do. ✅